src/vm/arm/stubs.cpp

```diff
@@ -534,13 +534,21 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
 
     do
     {
+#ifdef DACCESS_COMPILE
```

Comment:
Also, would you mind doing the same change for ARM64? It has the same problem (https://github.com/dotnet/coreclr/blob/master/src/vm/arm64/stubs.cpp#L323)